lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -56,6 +56,11 @@ internal sealed class DriverFolderPreparationHelper
         private readonly FileSets _fileSets;
         private readonly ISet<IConfigurationProvider> _driverConfigurationProviders;
 
+        /// <summary>
+        /// The folder in which we search for the client jar.
+        /// </summary>
+        private const string JarFolder = "./";
```

Comment:
In the manner of [JavaClientLauncher.cs](https://github.com/apache/reef/blob/3483e3d25a108934f118961dcf335410090b2d4b/lang/cs/Org.Apache.REEF.Client/Common/JavaClientLauncher.cs).

Reply:
Maybe add a comment to that effect?